sdk/src/transaction_context.rs

```diff
@@ -51,15 +56,94 @@ pub struct InstructionAccount {
 /// An account key and the matching account
 pub type TransactionAccount = (Pubkey, AccountSharedData);
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct TransactionAccounts {
+    accounts: Vec<RefCell<AccountSharedData>>,
+    // FIXXME: use bitfield
```

Comment:
Left over TODO

Reply:
I'm actually going to delete the comment. It's about using a bitfield instead of
Box<[bool]> but there's no need why this needs to be done in this branch and not
in master.